fapi-conformance-suite-configs/fapi-BR-rw-id2-with-mtls-PS256-PS256.json

```diff
@@ -0,0 +1,108 @@
+{
+    "alias": "keycloak",
+    "description": "FAPI-RW-ID2: Keycloak test with mtls client authentication (RequestObject:PS256/IDToken:PS256)",
```

Comment:
For FAPI-BR, is it OK to adapt FAPI-RW-ID2 instead of FAPI 1.0 Advanced Final?

Reply:
sure!